astropy/convolution/kernels.py

```diff
@@ -801,10 +801,9 @@ class Moffat2DKernel(Kernel2D):
     _is_bool = False
 
     def __init__(self, gamma, alpha, **kwargs):
-        self._model = models.Moffat2D((gamma - 1.0) / (np.pi * alpha * alpha),
+        self._model = models.Moffat2D((alpha - 1.0) / (np.pi * gamma * gamma),
```

Comment:
Might be good to reference this now.  I believe you're right, and suspect that maybe the swap came from confusion with the [wikipedia article](https://en.wikipedia.org/wiki/Moffat_distribution), which uses beta to represent alpha and alpha for gamma?